app/blog/views/route.ts

```diff
@@ -0,0 +1,8 @@
+import { db } from 'lib/planetscale';
+import { NextResponse } from 'next/server';
+
+export async function GET(request: Request) {
+  const data = await db.selectFrom('views').select(['slug', 'count']).execute();
+
+  return NextResponse.json({ data });
```

Comment:
Hey @leerob 
Line 7 will break the data.find method on `app/blog/view-counter.tsx` destructured from useSWR method. This returns a response with an object with key `data`.

We can use `return NextResponse.json(data);` instead.

Reply:
Thank you!